### ISSUE TITLE
add code lookup option

### ISSUE THREAD

**Issue by andres-lowrie** [koalaman/shellcheck#2413]
#### For new checks and feature suggestions
- [x] https://www.shellcheck.net/ (i.e. the latest commit) currently gives no useful warnings about this
- [x] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

Would be kinda cool if there was a "built in" way to get details on a error; something that perhaps read from the wiki 

eg:

```
shellcheck --code-details 1000
```

Would then output the info from the wiki perhaps 

```
`$` is not used specially and should therefore be escaped.

### Problematic code:

~~~sh
echo "$"
~~~

### Correct code:

~~~
echo "\$"
~~~

### Rationale:
`$` is special in double quotes, but there are some cases where it's interpreted literally:

1. Following a backslash: `echo "\$"`
2. In a context where the shell can't make sense of it, such as at the end of the string, (`"foo$"`) or before some constructs (`"$'foo'"`).

To avoid relying on strange and shell-specific behavior, any `$` intended to be literal should be escaped with a backslash.

Removed in [v0.3.3 - 2014-05-29](https://github.com/koalaman/shellcheck/blob/2e5c56b27034492134be1538c1e1e6a533ca791a/CHANGELOG.md#removed-1)
```

**Comment by Trysupe** [comment]
Not an implemented feature but a simple bash function can do the trick:

```bash
shellcheck_lookup(){
    [[ -z "$1" ]] && echo "No Issue provided" && return 1
    [[ ! "$1" =~ ^SC[0-9]{4}$ ]] && echo "Not a valid Issue" && return 1

    curl --url  "https://raw.githubusercontent.com/wiki/koalaman/shellcheck/$1.md" # Pipe this into your favourite md viewer
}
```

Usage: `shellcheck_lookup SC1118`

**Comment by andres-lowrie** [comment]
Awesome! Thank you much!
